example/ssd/dataset/iterator.py

```diff
@@ -185,6 +183,10 @@ def _data_augmentation(self, data, label):
         else:
             interp_methods = [cv2.INTER_LINEAR]
         interp_method = interp_methods[int(np.random.uniform(0, 1) * len(interp_methods))]
-        data = resize(data, self._data_shape, interp_method)
-        data = transform(data, self._mean_pixels)
+        data = mx.img.imresize(data, self._data_shape[0], self._data_shape[1], interp_method)
+        data = mx.nd.transpose(data, (2,0,1))
+        data = data.astype('float32')
+        data[0] -= self._mean_pixels[0]
+        data[1] -= self._mean_pixels[1]
+        data[2] -= self._mean_pixels[2]
```

Comment:
Ok. -= wasn't working because there was a bug. It's fixed by https://github.com/dmlc/mxnet/pull/4441